ceres.py

```diff
@@ -317,7 +317,9 @@ def readMetadata(self):
     :raises: :class:`CorruptNode`
     """
     try:
-      metadata = json.load(open(self.metadataFile, 'r'))
+      f = open(self.metadataFile, 'r')
+      metadata = json.load(f)
+      f.close()
```

Comment:
This could use the `with open` pattern to avoid the need to worry about closing the file

```
  with open(self.metadataFile, 'r') as f:
      metadata = json.load(f)
```